src/tools/JitOrderParser/jitOrderParser.cs

```diff
@@ -0,0 +1,333 @@
+                        foreach (OrderInfo matchOrderInfo in currList)
+                        {
+                            double diffScore = matchOrderInfo.perfScore - currRootOrderInfo.perfScore;
+                            int codeSizeDiff = matchOrderInfo.nativeHotBytes - currRootOrderInfo.nativeHotBytes;
+
+                            if (Math.Abs(diffScore) < 0.015)
+                            {
+                                if (codeSizeDiff == 0)
+                                {
+                                    Console.WriteLine("SAME: Root PerfScore of {2,7:F2} was the same as the other PerfScore for {0:x8} :: {1}",
+                                        currRootOrderInfo.methodHash, currRootOrderInfo.methodName, currRootOrderInfo.perfScore);
+                                }
+                                else if (Math.Abs(codeSizeDiff) < 6)
+                                {
+                                    Console.WriteLine("CLOSE: Root PerfScore was the same PerfScore of {2,7:F2} but had code size diff of {3} for {0:x8} :: {1}",
+                                        currRootOrderInfo.methodHash, currRootOrderInfo.methodName, matchOrderInfo.perfScore, codeSizeDiff);
+                                }
+                                else
+                                {
+                                    Console.WriteLine("DIFF: Same PerfScore of {2,7:F2} but had code size diff of {3} for {0:x8} :: {1}",
+                                        currRootOrderInfo.methodHash, currRootOrderInfo.methodName, currRootOrderInfo.perfScore, codeSizeDiff);
+                                }
+                            }
+                            else
+                            {
+                                double pctScore = diffScore / currRootOrderInfo.perfScore;
+                                // Is it within 2%
+                                if ((Math.Abs(pctScore) < 0.02) && (Math.Abs(codeSizeDiff) < 6))
+                                {
+                                    Console.WriteLine("CLOSE: Root PerfScore was differrent by {2,5:F2}% than the orig PerfScore of {3,7:F2}  for {0:x8} :: {1}",
+                                        currRootOrderInfo.methodHash, currRootOrderInfo.methodName, pctScore*100.0, matchOrderInfo.perfScore);
+                                }
+                                else
+                                {
+                                    Console.WriteLine("DIFF: Root PerfScore was differrent by {2,7:F2} than the other PerfScore for {0:x8} :: {1}",
+                                        currRootOrderInfo.methodHash, currRootOrderInfo.methodName, diffScore);
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        static void Main(string[] args)
+        {
+            int len = args.Length;
```

Comment:
This seems to be missing usage documentation

Reply:
This tool is still just a work in progress. I will move it over to jitutils and work on it some more.